test/github_pullrequest_concurrecy_test.go

```diff
@@ -160,5 +160,5 @@ func createGithubRepoCRDC(ctx context.Context, t *testing.T, ghprovider github.P
 }
 
 // Local Variables:
-// compile-command: "go test -tags=e2e -v -info TestGithubPullRequest$ ."
+// compile-command: "go test -tags=e2e -v -info TestGithubPullRequestWithConcurrency$ ."
```

Comment:
did you convert to emacs??? 😍

Reply:
nope 😅  I tried emacs before but .. was a bit tough to get used too.